src/ansiblelint/app.py

```diff
@@ -290,6 +290,8 @@ def report_summary(  # pylint: disable=too-many-branches,too-many-locals
             msg += f", and fixed {summary.fixed} issue(s)"
         msg += f" on {files_count} files."
 
+        msg += get_version_warning()
```

Comment:
Do you plan to make this optional ? Having something pinging github every day doesn't seem a good idea [I'm thinking of cases like air-gap users] but it's my personal opinion.

Reply:
In air-gap environments the warning will not appear, by design. But on all other cases it will tell user to update and not raise bugs. We had more than 2/3 reports on old version about stuff that was already fixed... so that drive this feature.

Reply:
Still, it would be nice to be able to disable it imho. I totally understand your aim but if I don't want to update for whatever reason / regression, it would be nice to be able to not get the message. Of course, for the exact reasons you're giving, it obviously should be enabled by default. Not the opposite.